beta-src/src/components/ui/WDInfoDisplay.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from "react";
+import {
+  Box,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  TableContainer,
+} from "@mui/material";
+import GameMode from "../../enums/GameMode";
+import Season from "../../enums/Season";
+import Ranking from "../../enums/Ranking";
+import Phase from "../../enums/Phase";
+
+/**
+ * game setting datas which would be passed to the component by parent component/ context/redux store
+ */
+
+interface WDInfoDisplayProps {
+  gameMode: GameMode;
+  gameTime: number | React.ReactNode;
+  phase: Phase;
+  potNumber: number;
+  rank: Ranking;
+  season: Season;
+  title: string;
+  year: number | string;
+}
+
+const tableCellStyles = {
+  border: "none",
+  fontSize: "0.7rem",
+  p: "0 5px 0 0",
+};
+
+const WDInfoDisplay: React.FC<WDInfoDisplayProps> = function ({
+  gameMode,
+  gameTime,
+  phase,
+  potNumber,
+  rank,
+  season,
+  title,
+  year,
+}) {
+  return (
+    <TableContainer>
+      <Table
+        aria-label="a dense table"
+        size="small"
+        sx={{
+          maxWidth: 250,
+        }}
+      >
+        <TableHead>
+          <TableRow>
+            <TableCell
+              sx={{
+                border: "none",
+                fontWeight: 600,
+                p: "2px 5px 5px 0",
+              }}
+            >
+              <Box
+                /**
+                 * by applying CSS properties, the title of the game would be able to wrap until the second line
+                 * for the second line display '...' to show it being cut-off
+                 */
+                sx={{
+                  display: "-webkit-box",
+                  lineHeight: "1.2rem",
+                  maxHeight: "2.4rem",
+                  overflow: "hidden",
+                  textOverflow: "ellipsis",
+                  wordWrap: "break-word",
+                  WebkitLineClamp: 2,
+                  WebkitBoxOrient: "vertical",
+                }}
+              >
+                {title}
+              </Box>
+            </TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          <TableRow>
+            <TableCell sx={tableCellStyles}>
+              Next phase in: {gameTime}, {phase},
```

Comment:
You will never print out the `gameTime` here as it is a unix timestamp. Rather you need to integrate the `formatTime` component implemented here:https://github.com/codazen/webDiplomacy/pull/31/files 

Please communicate with @sariacodazen if you have any questions about how to integrate this.